### PR TITLE
Fix comment preservation bug

### PR DESCRIPTION
**Pull request by uohzxela**

### CODE REVIEW

File: lib/elixir/test/erlang/tokenizer_test.erl [hunk 1]
```diff
@@ -84,7 +84,9 @@ float_test() ->
 comments_test() ->
   [{number, {1, 1, 2}, 1}, {eol, {1, 3, 4}}, {number, {2, 1, 2}, 2}] = tokenize("1 # Comment\n2"),
   [{number, {1, 1, 2}, 1}, {comment, {1, 3, 12}, "# Comment"},
-   {eol, {1, 12, 13}}, {number, {2, 1, 2}, 2}] = tokenize("1 # Comment\n2", [{preserve_comments, true}]).
+   {eol, {1, 12, 13}}, {number, {2, 1, 2}, 2}] = tokenize("1 # Comment\n2", [{preserve_comments, true}]),
+  [{comment,{1,1,10},"# Comment"}] = tokenize("# Comment", [{preserve_comments, true}]).
+
```
Comment (josevalim) [inline lib/elixir/test/erlang/tokenizer_test.erl]: Unnecessary extra new line.

File: lib/elixir/test/erlang/tokenizer_test.erl [hunk 2]
```diff
@@ -84,7 +84,9 @@ float_test() ->
 comments_test() ->
   [{number, {1, 1, 2}, 1}, {eol, {1, 3, 4}}, {number, {2, 1, 2}, 2}] = tokenize("1 # Comment\n2"),
   [{number, {1, 1, 2}, 1}, {comment, {1, 3, 12}, "# Comment"},
-   {eol, {1, 12, 13}}, {number, {2, 1, 2}, 2}] = tokenize("1 # Comment\n2", [{preserve_comments, true}]).
+   {eol, {1, 12, 13}}, {number, {2, 1, 2}, 2}] = tokenize("1 # Comment\n2", [{preserve_comments, true}]),
+  [{comment,{1,1,10},"# Comment"}] = tokenize("# Comment", [{preserve_comments, true}]).
```
Comment (whatyouhide) [inline lib/elixir/test/erlang/tokenizer_test.erl]: We need to have spaces after commas here:

```erlang
{comment, {1, 1, 10}, "# Comment"}
```

**Comment by uohzxela** [comment]
@josevalim @whatyouhide updated

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**